modules/rn_3_80.adoc

```diff
@@ -0,0 +1,148 @@
+[[rn-3-800]]
+== Version 3.8.0
+
+=== {productname}, Clair, and Quay Builder
+
+Added/Changed:
```

Comment:
Just curious if this is the subtitle you want to keep since it has a `/` in it. Could it be `Updates` since it captures both?

Reply:
Never much cared for added/changed. I'm going to look into changing it for the next release.